climpred/predictability_horizon.py

```diff
@@ -0,0 +1,60 @@
+import numpy as np
+import xarray as xr
+
+
+def _last_item_cond_true(cond, dim, shift=-1):
+    """Get the last item of condition ``cond`` which is True.
+
+    Args:
+        cond (xr.DataArray, xr.Dataset): Condition True means predictable. ``cond``
+            contains at least the dimension lead and checks for how many leads the
+            ``condition`` is true and hence skill is predictable.
+        dim (str): Dimension to check for condition == True over.
```

Comment:
Since the description for `cond` says that it needs to contain at least lead, do we need this? You could either just force this to always be over `lead` or you could say `dim='lead'` as the default and update the `cond` description to say ``cond`` must contain at least ``dim``